### PR TITLE
macOS: Install XQuartz

### PR DESCRIPTION
**Pull request by seisman**
XQuartz is at least needed by two software, SAC and CPS330.

I think it's better to install it at the start.

### CODE REVIEW

File: source/computer/macos-setup.rst [hunk 1]
```diff
@@ -199,6 +199,14 @@ git
    目录下。在环境变量 **PATH** 中，:file:`/usr/local/bin/` 一般在 :file:`/usr/bin/` 前。
    因此，我们使用的 ``git`` 其实是通过 Homebrew 安装的版本。
 
+X11
+^^^
+
+X11 是 UNIX 系统下常用的一种视窗系统。某些软件（如 SAC 和 CPS330）需要使用它。
```
Comment (core-man) [inline source/computer/macos-setup.rst:205]:
```suggestion
X11（也称 X Window System）是 UNIX 系统下常用的一种视窗系统。某些软件（如 SAC 和 CPS330）需要使用它。
```

**Comment by core-man** [comment]
I cannot use X11 after I install XQuartz.
```
$ sac
$ fg seis
$ ppk
```

So I restart macOS, and then it works.

File: source/computer/macos-setup.rst [hunk 2]
```diff
@@ -199,6 +199,14 @@ git
...
    因此，我们使用的 ``git`` 其实是通过 Homebrew 安装的版本。
 
+X11
+^^^
+
+X11（也称 X Window System）是 UNIX 系统下常用的一种视窗系统。某些软件（如 SAC 和 CPS330）需要使用它。
+在 macOS 下，需要安装 `XQuartz <https://www.xquartz.org/>`__::
+
+   $ brew install --cask xquartz
+
```
Comment (seisman) [inline source/computer/macos-setup.rst:209]:
```suggestion
X11（也称 X Window System）是 UNIX 系统下常用的一种视窗系统。某些软件（如 SAC 和 CPS330）需要使用它。
在 macOS 下，需要安装 `XQuartz <https://www.xquartz.org/>`__::

   $ brew install --cask xquartz

需要注意，可能需要重启电脑后 X11 才能生效。建议等配置完 macOS 后再重启。
```

**Review by core-man — APPROVED** [review]

File: source/computer/macos-setup.rst [hunk 3]
```diff
@@ -199,6 +199,15 @@ git
...
+需要注意，可能需要重启电脑后 X11 才能生效。建议等配置完 macOS 后再重启。
```
Comment (seisman) [inline source/computer/macos-setup.rst:210]:
```suggestion
需要注意，可能需要重启电脑后 X11 才能生效。建议等配置完 macOS 后再重启。

```

**Pull request merged**